shogun-lib/pom.xml

```diff
@@ -231,11 +231,6 @@
       <artifactId>HikariCP</artifactId>
       <scope>test</scope>
     </dependency>
-      <dependency>
-          <groupId>org.reflections</groupId>
-          <artifactId>reflections</artifactId>
-          <version>0.9.12</version>
-      </dependency>
```

Comment:
This should be readded (without the version).

Reply:
this is now in the pom.xml on top level

Reply:
nevermind, i add it